pallets/delegation/src/lib.rs

```diff
@@ -211,32 +219,19 @@ pub mod pallet {
 			parent_id: Option<DelegationNodeIdOf<T>>,
 			delegate: DelegatorIdOf<T>,
 			permissions: Permissions,
-			delegate_signature: did::DidSignature,
+			delegate_signature: Vec<u8>,
```

Comment:
maybe use the associate type  `DelegationSignatureVerification::Signature` here?

Reply:
Fixed in https://github.com/KILTprotocol/mashnet-node/pull/179/commits/6d1d7a766b55e1f2ab874caf87090b7db2e6e300.